src/pages/resources/reference/push-notifications/templates/tabs/timer.md

```diff
@@ -77,7 +171,7 @@ Below is a sample of what a payload might look like for a notification using the
 
 ## Usage recommendations
 
-The tables below contain guidelines for your push notification content. **These recommendations help your text display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
+The tables below contain guidelines for your push notification content. **These recommendations help your text and images display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
```

Comment:
Similar comment for the it is recommended as above: "... guidelines only - you should still test a notification..."